.github/workflows/run_pr_tests.yml

```diff
@@ -163,3 +164,33 @@ jobs:
           build_dir: ${{ github.workspace }}/build_offline
           c_compiler: 'cl.exe'
           cxx_compiler: 'cl.exe'
+
+############### JOB   mr-ubuntu-gcc-x86_64-riscv-fp16-cl3-0-unitcl_vecz:
+
+  mr-ubuntu-gcc-x86_64-riscv-fp16-cl3-0-unitcl_vecz:
```

Comment:
Do we want to drop the "mr" as the job name prefix and replace with "run"?
Also other jobs use underscores as separators in the name - not fussy about that but I guess we should be consistent.

Reply:
This is just moving your job across, i think it helps to cross-correlate for now.